merkle_tree/Makefile

```diff
@@ -0,0 +1,5 @@
+run:
+	cargo run
+
+test:
+	cargo test
```

Comment:
Missing a new line here. Careful with this, since some unix tools rely on a newline at the end of the file to work properly (Like diff).